### DISCUSSION TITLE
Angular Scully Preset does not access Chromium

### DISCUSSION THREAD

**Discussion by fabioemoutinho** [vercel/next.js#34896] [Help] [answered]
I was not able to make the Angular Scully preset work with default configuration, when the Scully build step starts, it fails to run Chromium.
Anyone knows what the problem might be?

`Puppeteer cannot find or launch the browser. (by default chrome)`

**Comment by fabioemoutinho** [comment, marked as answer]
adding the following to scully.[project].config.ts worked:

```
puppeteerLaunchOptions: {
  args: ['--no-sandbox'],
},
```

**Reply by sefatanam** [reply]
not working for me. The same error appear with warning like this,
`
  Puppeteer old Headless deprecation warning:
    In the near future `headless: true` will default to the new Headless mode
    for Chrome instead of the old Headless implementation. For more
    information, please see https://developer.chrome.com/articles/new-headless/.
    Consider opting in early by passing `headless: "new"` to `puppeteer.launch()`
    If you encounter any bugs, please report them to https://github.com/puppeteer/puppeteer/issues/new/choose.
`
